modules/cluster-logging-updating-prereqs.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
```

Comment:
I'm generally confused regarding this entire section. We  need to discuss

Reply:
Sorry. Forgot about this. It has become a repository for notes. I will remove.